Packs/SekoiaXDR/Integrations/SekoiaXDR/SekoiaXDR.py

```diff
@@ -610,7 +626,188 @@ def fetch_incidents(
+        reopen_incident: Indicates whether to reopen the corresponding XSOAR incident if the alert
+            has been reopened on Sekoia's end.
+    Returns:
+        GetRemoteDataResponse: The Response containing the update alert to mirror and the entries
+    """
+
+    demisto.debug("#### Entering MIRRORING IN - get_remote_data_command ####")
+
+    parsed_args = GetRemoteDataArgs(args)
+    alert = client.get_alert(alert_uuid=parsed_args.remote_incident_id)
+
+    alert_short_id, alert_status = alert["short_id"], alert["status"]["name"]
+    last_update = arg_to_timestamp(
+        arg=parsed_args.last_update, arg_name="lastUpdate", required=True
+    )
+    alert_last_update = arg_to_timestamp(
+        arg=alert.get("updated_at"), arg_name="updated_at", required=False
+    )
+
+    demisto.debug(
+        f"Alert {alert_short_id} with status {alert_status} : last_update is {last_update} , alert_last_update is {alert_last_update}"  # noqa: E501
+    )
+
+    entries = []
+
+    # Add the events to the alert
+    if mirror_events and alert["status"]["name"] not in ["Closed", "Rejected"]:
+        earliest_time = alert["first_seen_at"]
+        lastest_time = "now"
+        term = f"alert_short_ids:{alert['short_id']}"
+        interval_in_seconds = INTERVAL_SECONDS_EVENTS
+        timeout_in_seconds = TIMEOUT_EVENTS
+
+        args = {
+            "earliest_time": earliest_time,
+            "lastest_time": lastest_time,
+            "query": term,
+            "interval_in_seconds": interval_in_seconds,
+            "timeout_in_seconds": timeout_in_seconds,
+        }
+        events = search_events_command(args=args, client=client)
+        alert["events"] = events.outputs  # pylint: disable=E1101
+
+    # Add the kill chain information to the alert
+    if mirror_kill_chain and alert["kill_chain_short_id"]:
+        try:
+            kill_chain = client.get_kill_chain(
+                kill_chain_uuid=alert["kill_chain_short_id"]
+            )
+            alert["kill_chain"] = kill_chain
+        except Exception as e:
+            # Handle the exception if there is any problem with the API call
+            demisto.debug(f"Error fetching kill_chain : {e}")
+
+    # This adds all the information from the XSOAR incident.
+    demisto.debug(
+        f"Alert {alert_short_id} with status {alert_status} have this info updated: {alert}"
+    )
+
+    investigation = demisto.investigation()
+    demisto.debug(f"The investigation information is {investigation}")
+
+    incident_id = investigation["id"]
+    incident_status = investigation["status"]
+
+    demisto.debug(
+        f"The XSOAR incident is {incident_id} with status {incident_status} is being mirrored with the alert {alert_short_id} that have the status {alert_status}."  # noqa: E501
+    )
+
+    # Close the XSOAR incident using mirroring
+    if (
+        (close_incident)
+        and (alert_status in ["Closed", "Rejected"])
+        and (investigation["status"] != 1)
```

Comment:
```suggestion
        and (investigation["status"] != 2)
```
When the status of the incident isn't 1, there could be two situations:
1. The incident was already closed. In this case, the incident status will be 2.
2. The incident wasn't activated yet (pending), and the incident status will be 0.